go.mod

```diff
@@ -1,96 +1,95 @@
 module github.com/uw-labs/substrate
 
-go 1.18
+go 1.19
 
 require (
-	github.com/Shopify/sarama v1.29.1
+	github.com/Shopify/sarama v1.38.1
```

Comment:
👍 bet lots of bug fixes since

Reply:
Nice. Also I think that before we actually merge this, we should let it run for some time in a team cluster in dev and see how it goes.

Reply:
Will leave it running with one of our services in dev

Reply:
So far there doesn't seem to be any issues publishing & consuming through proximo after this change